app/controllers/application_controller.rb

```diff
@@ -1,7 +1,18 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :null_session
+  skip_before_action :verify_authenticity_token
+
   rescue_from ActiveRecord::RecordNotFound, :with => :record_not_found
 
   def record_not_found
     head :not_found
   end
+
+  def create_dto_from_body(class_dto)
+    return class_dto::create(request.body.read)
+  end
+
+  def create_dto_from_params(class_dto)
+    return class_dto::create(request.params)
+  end
```

Comment:
Seems to be a good idea, but I would like to see how it works.

Reply:
It's not working. I started working on something automatic but it was taking to much time so I decided to implement easier version with passing parameters to Dto manually.